messaging/build.gradle

```diff
@@ -61,7 +61,7 @@ android {
 dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.6.0'
+    implementation 'androidx.core:core-ktx:1.3.2'
```

Comment:
I reverted this just to avoid any issues. I agree it would be nice to upgrade this stuff just to stay on a recent version, but it's not required for this PR. If you want, you can log a ticket for it and we can tackle it separately.